docker-compose.yml

```diff
@@ -203,6 +205,7 @@ services:
   #
   db:
     image: pgautoupgrade/pgautoupgrade:17-bookworm
+    shm_size: 5gb
```

Comment:
This was necessary because postgres processes would run out of memory and result in the following error:
```
ERROR:  could not resize shared memory segment "/PostgreSQL.1684934532" to 4194304 bytes: No space left on device
```